README.md

```diff
@@ -222,34 +222,19 @@ An runtime config automatically expanded by Lighter might look like
 ```
 
 ## Container Startup Sequence
-Docker images should embed the `secretary` executable. Call it at container startup to decrypt
-environment variables, before starting the actual service.
+Docker images should embed the `secretary` executable. Use it as an entrypoint
+with `secretary exec --". It will then decrypt all command line arguments and
+environment variables.
 
 *Dockerfile*
 ```
 # Install secretary
 ENV SECRETARY_VERSION x.y.z
 RUN curl -fsSLo /usr/bin/secretary "https://github.com/meltwater/secretary/releases/download/${SECRETARY_VERSION}/secretary-`uname -s`-`uname -m`" && \
     chmod +x /usr/bin/secretary
-```
-
-Container startup examples
-```
-#!/bin/sh
-set -e
-
-# Decrypt secrets
-if [ "$SERVICE_PUBLIC_KEY" != "" ]; then
-    SECRETS=$(secretary decrypt -e --service-key=/service/keys/service-private-key.pem)
```

Comment:
Hmm... it seems there is support already here: https://github.com/meltwater/secretary/pull/19/files#diff-7ddfb3e035b42cd70649cc33393fe32cR162 (line 162). However, we do remove the logic of "by default use /service/keys/service-private-key.pem if it exists".